contribution/development_guide.md

```diff
@@ -107,6 +162,52 @@
 
       Please ensure that the steps to setup K8s is followed so as to resolve any open dependencies.
```

Comment:
Please ensure that the steps to set up K8s are followed so as to resolve any open dependencies.